addons/godot-firebase/dynamiclinks/dynamiclinks.gd

```diff
@@ -34,7 +34,7 @@ enum Requests {
 func _set_config(config_json: Dictionary) -> void:
     _config = config_json
     _request_list_node = HTTPRequest.new()
-    _request_list_node.connect("request_completed", self, "_on_request_completed")
+    _request_list_node.request_completed.connect(Callable(self, "_on_request_completed"))
```

Comment:
I would rather see this as:

`_request_list_node.request_completed.connect(_on_request_completed)`